YarnSpinnerTests/Test.cs

```diff
@@ -101,13 +102,26 @@ public void TestIndentation()
 		}
 
 		[Test()]
+		public void TestVariableStorage()
+		{
+			storage.Clear ();
+
+			var path = System.IO.Path.Combine ("TestCases", "VariableStorage.node");
+			dialogue.LoadFile (path);
+
+			foreach (var result in dialogue.Run()) {
+				HandleResult (result);
+			}
+		}
+
+		// [Test()]
```

Comment:
A lot of the  `Test` attributes have been commented out - were you encountering errors in the unit tests?